tests/mgr/test_restful_operations_pool_get.py

```diff
@@ -0,0 +1,85 @@
+import json
+from cli.cephadm.cephadm import CephAdm
+from api.restful.pool import Pool
+from cli.exceptions import OperationFailedError
+from utility.log import Log
+
+LOG = Log(__name__)
+
+
+# Request header
+HEADER = {
+    "Accept": "application/vnd.ceph.api.v1.0+json",
+    "Content-Type": "application/json",
+}
+
+# Missing config parameter err msg
+CONFIG_ERR_MSG = "Mandatory config parameter '{}' not provided"
+
+
+# Mandatory test config exception
+class ConfigNotFoundError(Exception):
+    pass
+
+
+def validate_configs(config):
+    """Validate test config parameters"""
+    username = config.get("username")
+    if not username:
+        raise ConfigNotFoundError(CONFIG_ERR_MSG.format("username"))
+
+    endpoint = config.get("endpoint")
+    if not endpoint:
+        raise ConfigNotFoundError(CONFIG_ERR_MSG.format("endpoint"))
+
+    status_code = config.get("status_code")
+    if not status_code:
+        raise ConfigNotFoundError(CONFIG_ERR_MSG.format("status_code"))
+
+
+def run(ceph_cluster, **kw):
+    config = kw.get("config", {})
+    validate_configs(config)
+
+    # Get the MGR node
+    node = ceph_cluster.get_nodes(role="mgr")[0]
+    mgr_ip = node.ip_address
+
+    # Read the test configs
+    username = config.get("username")
+    endpoint = config.get("endpoint")
+    status_code = config.get("status_code")
```

Comment:
@sayaleeraut, you are reading test configs and validating them in validate_configs method. Is it not good to read and validate test configs together?